### DISCUSSION TITLE
Ender 3 V2 with E3V2-BLTouch-3x3-v4.2.2.bin not deploying

### DISCUSSION THREAD

**Discussion by RobertD502** [Jyers/Marlin#564] [Q&A] [answered]
Hello, just wondering if anyone else has ran into this problem. Whenever choosing Auto home from the menu, the BLTouch remains red and never deploys. Once at the center, the hotend proceeds to go down as it should (without the BLTouch deployed and with red LED on) and I have to stop it as it will crash into the bed. The BLTouch is wired into the dedicated port. I have attached the BLTouch that I am using. Could it be a clone requiring extra modifications in the firmware? ![image](https://user-images.githubusercontent.com/52541649/115801978-ff20b200-a3ab-11eb-80eb-548e3af89795.jpeg)

**Comment by christianjou** [comment, marked as answer]
Hi, check your connection on the mother board

**Reply by RobertD502** [reply]
Securely in there and wired correctly.

**Reply by gothcha** [reply]
It appears to be a real one. Does it power up correctly? It should deploy twice and flash, if not it isn’t wired correctly.

**Reply by ryankesavan** [reply]
apologies in advanced, but this should give you an idea of what to search for, for a better guide, but I had a similar issue, you probably need to adjust the probe screw on the top. 

if you have the official metal mount you should be able to access it with an allen key. loosen it so the screw comes up toward the metal and away from the probe and the probe should drop.
mine would sporadically deploy but once i adjusted the screw it operated correctly

**Reply by DeNohvo** [reply]
> apologies in advanced, but this should give you an idea of what to search for, for a better guide, but I had a similar issue, you probably need to adjust the probe screw on the top.
> 
> if you have the official metal mount you should be able to access it with an allen key. loosen it so the screw comes up toward the metal and away from the probe and the probe should drop.
> mine would sporadically deploy but once i adjusted the screw it operated correctly

What screw are you referring too?

**Reply by ryankesavan** [reply]
its a small black set screw.  its directly on top of the probe, if you remove it completely, you can remove the probe to replace it.

**Reply by ryankesavan** [reply]
https://www.reddit.com/r/3Dprinting/comments/ds2d85/thought_my_bltouch_was_broken/

![image](https://user-images.githubusercontent.com/58707083/116033228-0acfdb00-a62f-11eb-9c1c-3d1ceaa284fa.png)
